frontend/src/console/components/ConsolePage/ConsolePage.tsx

```diff
@@ -115,6 +116,18 @@ export class ConsolePageComponent extends React.PureComponent<Props> {
       return this.props.children;
     };
 
+    if (this.props.error === 'Not allowed for staff users') {
```

Comment:
As mentioned elsewhere, I would prefer if we could match on an error code instead of a full error message.